cmd/libs/go2idl/import-boss/generators/import_restrict.go

```diff
@@ -132,27 +131,6 @@ type importRuleFile struct{}
 
 func (importRuleFile) AssembleFile(f *generator.File, path string) error {
 	return nil
-
-	// If the file exists, populate its current imports. This is mostly to help
```

Comment:
This is useful if you want to see the list of imports that your rule will run against.